libs/blocks/tabs/tabs.js

```diff
@@ -84,89 +87,110 @@ function initTabs(elm, config, rootElem) {
   if (config) configTabs(config, rootElem);
 }
 
-let initCount = 0;
+const handleDeferredImages = (block) => {
+  const loadLazyImages = () => {
+    const images = block.querySelectorAll('img[loading="lazy"]');
+		  images.forEach((img) => {
+		    /* c8 ignore next */
+		    img.removeAttribute('loading');
+		  });
+		  document.removeEventListener('milo:deferred', loadLazyImages, true);
+  };
+  document.addEventListener('milo:deferred', loadLazyImages, true);
```

Comment:
Just realized we can probably use `once` so we don't have to `removeEventListener`.
```suggestion
  document.addEventListener('milo:deferred', loadLazyImages, { once: true, capture: true });
```